File: Disertatie_backend/DTO/Anime/AnimeCard.cs
using Disertatie_backend.Entities.Anime;
using MongoDB.Bson;

namespace Disertatie_backend.DTO.Anime
{
    public class AnimeCard
    {
        public string Title { get; set; }
        public string Popularity { get; set; }
        public ObjectId Id { get; set; }
        public double? Score { get; set; }
        public string Image { get; set; }
        public string Year { get; set; }
    }
}

File: ClientApp/src/app/_models/animeCard.ts
import { Images } from "model/images";
import { ObjectId } from "model/objectId";

export interface AnimeCard{
    id: ObjectId;
    Mal_id: number;
    fullTitle: string;
    year: number;
    images?: Images;
    score: number;
    title: string;
    popularity: number;
}